src/utils/carousel.ts

```diff
@@ -69,36 +70,37 @@ export function checkEventType(typeId: number) {
 }
 
 export function createWidget(event: Event) {
+  const { t } = useTranslation()
```

Comment:
AH! trovato! mi sembrava di aver visto un hook usato fuori da un componente ahah
banalissimo in realtà, basta importare direttamente da i18n
```ts
import { i18n } from "../locales/i18n"

const { t } = i18n